latest/docs-ref-conceptual/install-azure-cli.md

```diff
@@ -26,28 +26,28 @@ For information about the latest release, see the [release notes](release-notes-
 
 ## macOS
 
-> [!WARNING]
-> The Homebrew formula for the Azure CLI, `azure-cli`, is currently out of date and will install a previous version.
+1. Install `curl` via [homebrew](https://brew.sh/).
+
+   ```bash
+   brew install curl
```

Comment:
Note that `curl` is still available via homebrew, however: http://braumeister.org/formula/curl

There have also been verbatims left by users indicating that `curl` was not installed on their local machine, which have recent dates (2017) on them. I will need access to a clean macOS install/VM in order to test.